docs/stdlib/numbers.rst

```diff
@@ -446,11 +464,11 @@ from :eql:type:`str` and :eql:type:`json`.
 
     :index: modulo mod division
 
-    Remainder from division (modulo).
+    Retrieves the remainder from a division operation. (modulo)
 
-    This is the remainder from floor division. Just as is
-    the case with :eql:op:`//<floordiv>` the result type of the
-    remainder operator corresponds to the operand type:
+    This is also the remainder from floor division. Just as is
+    the case with the :eql:op:`//<floordiv>` operator, the resulting type of
+    the remainder operator corresponds to the left operand's type:
```

Comment:
`select 5 % 1.0 is typeof 5;` gives me `false` which leads me to believe this isn't entirely accurate. The original phrasing doesn't ring true either given that. @vpetrovykh, would it be accurate to say "the result type from the remainder operator is **derived from** the types of its operands?"